test/azure/Expected/AcceptanceTests/Lro/lro/aio/operations_async/_lro_retrys_operations_async.py

```diff
@@ -139,8 +142,17 @@ def get_long_running_output(pipeline_response):
         if polling is True: polling_method = AsyncARMPolling(lro_delay,  **kwargs)
         elif polling is False: polling_method = AsyncNoPolling()
         else: polling_method = polling
-        return await async_poller(self._client, raw_result, get_long_running_output, polling_method)
```

Comment:
What did we used to be "awaiting" here?

Reply:
It was equivalent of blocking call to get the result (it's why there was no begin_ prefix). So here we were waiting for the result